src/yoroi-wallets/hooks/index.ts

```diff
@@ -119,6 +119,11 @@ export const useUtxos = (wallet: YoroiWallet) => {
   return wallet.utxos
 }
 
+export const useAssets = (wallet: YoroiWallet): RemoteAsset[] => {
```

Comment:
`RemoteAsset` is a hack type because the backend team refuses to type anything
dont use this type
why do you need a separate hook for this?